src/components/slider/slider.component.html

```diff
@@ -1,11 +1,12 @@
 <div class="track"
      #track
+     [class.disabled]="disabled"
```

Comment:
This should not be needed since the parent element has a disabled class as well, you can use that in the selector instead.